src/components/TodoItem.js

```diff
@@ -0,0 +1,13 @@
+import React from "react"
+
+class TodoItem extends React.Component {
+  render() {
+    return <li><input
+    type="checkbox"
+    checked={this.props.todo.completed}
+    onChange={() => this.props.handleChangeProps(this.props.todo.id)}
+  /> {this.props.todo.title}</li>
+  }
+}
+
+export default TodoItem
```

Comment:
Hello, it would be a great idea to end this line with a semi-colon. Kindly have a look at list of [best practices for JavaScript](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md) for a refresher.